### ISSUE TITLE
[Bug] Time range as invalid float Unix timestamp

### ISSUE THREAD

**Issue by samcarey** [eclipse-zenoh/zenoh-backend-influxdb#53]
### Describe the bug

The [get](https://github.com/eclipse-zenoh/zenoh-backend-influxdb/blob/fb579c3eb9a8baa7df30a2f51782ded218d1cfff/v2/src/lib.rs#L593) function uses the [range](https://docs.influxdata.com/flux/v0/stdlib/universe/range/#start) flux function using Unix timestamps, but the [start](https://docs.influxdata.com/flux/v0/stdlib/universe/range/#start)/stop parameters must be integers when used as Unix timestamps, while the code tries to send floats with a decimal, which causes an error:
```console
[2024-01-04T21:36:37Z DEBUG zenoh_backend_influxdb2] Get Some("stored/test") with Influx query:from(bucket: "test")
                                                |> range(start: 1704404158.244, stop: 1704404197)
                                                |> filter(fn: (r) => r._measurement == "stored/test")
                                                |> filter(fn: (r) => r["kind"] == "PUT")
                                             in InfluxDBv2 storage
[2024-01-04T21:36:37Z ERROR zenoh_backend_influxdb2] Couldn't get data from database test in InfluxDBv2 storage with error: HTTP request returned an error: 400 Bad Request, `{"code":"invalid","message":"error calling function \"range\" @2:48-2:94: value is not a time, got float"}`
```

I think this could be fixed by either
1. Rounding the timestamp to the nearest second (less than ideal) or
2. Converting to RFC3339 format, which might (I haven't tried it) allow sub-second precision.

### To reproduce

1. Start zenoh with the influxdb2 plugin
2. Try a query with a specific absolute time range. See below for example.
```rust
use async_std::stream::StreamExt;
use chrono::{SecondsFormat, Utc};
use std::time::Duration;
use zenoh::prelude::r#async::*;

#[async_std::main]
async fn main() {
    let mut config = config::default();
    config
        .connect
        .endpoints
        .push("tcp/localhost:7447".parse().unwrap());
    let session = zenoh::open(config).res().await.unwrap();
    let before = Utc::now();
    let mut query_value = None;
    while query_value.is_none() {
        async_std::task::sleep(Duration::from_secs(1)).await;
        let time = before.to_rfc3339_opts(SecondsFormat::Millis, true);
        println!("{}", time);
        query_value = session
            .get(format!("stored/test?_time=[{time}..]"))
            .res()
            .await
            .unwrap()
            .stream()
            .map(|r| r.sample.unwrap())
            .next()
            .await;
    }
}
```
Once the above connects and attempts a query, you should see the above error in the Zenoh console.

### System info

- Windows 11, AMD64
- Zenoh commit 788172b0360caa38db24ec2d238734a2ebf40d32
- zenoh_backend_influxdb2 commit fb579c3eb9a8baa7df30a2f51782ded218d1cfff

**Comment by snehilzs** [comment]
Hi @samcarey ,
Thanks for bringing this to our attention. We will be adding a bug fix for this soon.

**Comment by JEnoch** [comment]
Fixed by #59
